### PR TITLE
Feature/launch pdf terms

### PR DESCRIPTION
**Pull request by SERDUN**

### CODE REVIEW

File: ios/Runner/Info.plist [hunk 1]
```diff
@@ -67,5 +67,9 @@
 		<true/>
 		<key>UIStatusBarHidden</key>
 		<false/>
+		<key>LSApplicationQueriesSchemes~ipad</key>
```
Comment (ycherniavskyi) [inline ios/Runner/Info.plist:70]: What is the reason to add `~ipad` to this key?

File: ios/Runner/Info.plist [hunk 2]
```diff
@@ -67,5 +67,9 @@
...
+		<array>
+			<string>https</string>
```
Comment (ycherniavskyi) [inline ios/Runner/Info.plist:72]: Let's also add `http` value.

File: android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -15,6 +15,13 @@
     <uses-permission android:name="android.permission.READ_CONTACTS"/>
     <uses-permission android:name="android.permission.WRITE_CONTACTS"/>
     <uses-permission android:name="android.permission.GET_ACCOUNTS"/>
+    <queries>
+        <intent>
+            <action android:name="android.intent.action.VIEW" />
+            <category android:name="android.intent.category.BROWSABLE" />
+            <data android:scheme="https" />
+        </intent>
```
Comment (ycherniavskyi) [inline android/app/src/main/AndroidManifest.xml:23]: Let's also add intent for `http` scheme.

File: lib/features/settings/view/settings_screen.dart [hunk 4]
```diff
@@ -210,4 +217,10 @@ class SettingsScreen extends StatelessWidget {
       child: scaffold,
     );
   }
+
+  void _launchUrl(BuildContext context, Uri uri) async {
```
Comment (ycherniavskyi) [inline lib/features/settings/view/settings_screen.dart:221]: The `context` parameter is unnecessary here.
And also let's give this method a more specific name - something like `_launchUrlWithDefaultPlatformViewer`.

File: lib/features/settings/view/settings_screen.dart [hunk 5]
```diff
@@ -138,7 +139,13 @@ class SettingsScreen extends StatelessWidget {
               title: Text(context.l10n.settings_ListViewTileTitle_termsConditions),
               trailing: const Icon(Icons.navigate_next),
               onTap: () {
-                context.pushNamed(MainRoute.settingsTermsConditions);
+                final uri = Uri.parse(EnvironmentConfig.APP_TERMS_AND_CONDITIONS_URL);
+
+                if (uri.path.endsWith('.pdf')) {
+                  _launchUrl(context, uri);
+                } else {
+                  context.pushNamed(MainRoute.settingsTermsConditions);
```
Comment (ycherniavskyi) [inline lib/features/settings/view/settings_screen.dart:147]: It seems that we could make some refactoring with this PR, and pass the "terms and conditions URL" as a routing parameter to the `initialUri` property of the `TermsConditionsScreen` widget.

**Review by ycherniavskyi — CHANGES_REQUESTED** [review]

File: ios/Runner/Info.plist (hunk 1 above)
Comment (SERDUN) [reply ios/Runner/Info.plist:70]: thank you, I can't understand the world it took, I'll delete it

File: lib/features/settings/view/settings_screen.dart (hunk 5 above)
Comment (SERDUN) [reply lib/features/settings/view/settings_screen.dart:147]: okay ,  good point.
What about key ''initialUrl', should we move it in const ? but i not sure where it better to keep  consts.dart or routes.dart

Comment (SERDUN) [reply lib/features/settings/view/settings_screen.dart]: tried to bring out the hard coded values in route queries keys

**Review by ycherniavskyi — APPROVED** [review]

**Pull request merged**